pde_superresolution/integrate.py

```diff
@@ -111,6 +111,22 @@ def __call__(self, t: float, y: np.ndarray) -> np.ndarray:
     return self.equation.finalize_time_derivative(t, time_derivative)
 
 
+class WENODifferentiator(Differentiator):
+  """Calculate derivatives using a 5th order WENO.py method."""
+
+  def __init__(self, equation: equations.Equation, alpha: float = 1.0, **kwargs):
+    assert isinstance(equation, equations.ConservativeBurgersEquation)
+    self.equation = equation
+    self.weno = WENO(dx=equation.grid.reference_dx)
```

Comment:
@shoyer Should this be `dx=equation.grid.solution_dx`?

Reply:
Yes, it definitely should be `equation.grid.solution_dx`